### PR TITLE
[docs] Cloud Troubleshooting

### PR DESCRIPTION
**Pull request by ddhodge**
Troubleshooting page for Cloud issues

@netlify /latest/yugabyte-cloud/cloud-troubleshoot/

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Be sure to encode any special characters in your connection string.
+
+### VPC networking
+
+If you have set up a VPC network and are unable to connect, verify the following.
+
+#### VPC status is Failed
+
+If you are unable to successfully create the VPC, contact Yugabyte Support.
```
Comment (chrisckchang) [inline docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md:54]: @ddhodge can we hyperlink "contact Yugabyte Support" with either `mailto:support@yugabyte.com` or `https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431`

File: docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+```output
+ysqlsh: could not connect to server: Operation timed out
+    Is the server running on host "4477b44e-4f4c-4ee4-4f44-f44e4abf4f44.aws.ybdb.io" (44.144.244.144) and accepting
+    TCP/IP connections on port 5433?
+```
+
+If you are trying to connect to a cluster from your local computer, add your computer to the cluster [IP allow list](../cloud-basics/add-connections/). If your IP address has changed, add the new IP address.
+
+If you have a VPC configured, add one or more IP addresses from the peered VPN to the cluster [IP allow list](../cloud-basics/add-connections/).
```
Comment (Arnav15) [inline docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md:30]: VPN -> VPC

File: docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+If you are unable to successfully create the VPC, contact [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431).
+
+#### Peering connection status is Pending
+
+A peering connection status of _Pending_ indicates that you need to configure your cloud provider to accept the connection. Refer to [Create a peering connection](../cloud-vpcs/cloud-add-peering/#peer-aws).
+
+#### Peering connection status is Expired (AWS only)
+
+The peering request was not accepted. Retry the peering connection.
```
Comment (Arnav15) [inline docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md:62]: Recreate instead of retry?

**Review by Arnav15 — APPROVED** [review]
Just few nits. Looks good otherwise.

**Pull request merged**